spec/spec_helper.rb

```diff
@@ -17,9 +21,11 @@
 Dir[File.join(File.dirname(__FILE__), "support/**/*.rb")].each {|f| require f }
 
 require 'spree/testing_support/factories'
+require_relative '../spec/factories'
```

Comment:
I needed to add some factories that weren't included in the spree/testing_support factories to support testing the track_inventory flag feature, particularly around stock locations and inventory units.